client/state/automated-transfer/selectors.js

```diff
@@ -32,19 +32,33 @@ export const getAutomatedTransferStatus = compose(
 /**
  * Helper to get eligibility state from local transfer state sub-tree
  *
- * @param {Object} state automated transfer state sub-tree for a site
+ * @param {Object} atatState automated transfer state sub-tree for a site
```

Comment:
We know and love our :atat:, but I wonder if someone might come along later and think this is a typo. Maybe just `atState`?